src/collect-lp.ts

```diff
@@ -236,11 +248,23 @@ export class LpCollector {
         logger.debug(
           `Collecting LP reward as collateral. pool ${this.pool.name}`
         );
+        
+        // Get LP balance before the transaction
+        const signerAddress = await this.signer.getAddress();
+        const { lpBalance: lpBalanceBefore } = await bucket.getPosition(signerAddress);
+        
         await bucketRemoveCollateralToken(
           bucket,
           this.signer,
           collateralToWithdraw
         );
+        
+        // Get LP balance after the transaction
+        const { lpBalance: lpBalanceAfter } = await bucket.getPosition(signerAddress);
+        
+        // Calculate the actual LP used
+        const actualLpUsed = lpBalanceBefore.sub(lpBalanceAfter);
```

Comment:
Some comment as above; this variable is not needed.

Reply:
agree